packages/plugin-snowplow/build.gradle

```diff
@@ -0,0 +1,87 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform'
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android-extensions'
+apply from: rootProject.file('gradle/publish-mpp.gradle')
+apply from: rootProject.file('gradle/android-library.gradle')
+
+kotlin {
+    android() {
+        publishLibraryVariants("release", "debug")
+        publishLibraryVariantsGroupedByFlavor = true
+    }
+
+    targets.all {
+        compilations.all {
+            kotlinOptions {
+                // jvmTarget = "1.8"
+                allWarningsAsErrors = false
+            }
+        }
+    }
+
+    sourceSets {
+        // COMMON
+        commonMain {
+            dependencies {
+                implementation kotlin('stdlib-jdk8')
+                implementation project(":packages:sdk-core")
+            }
+        }
+        commonTest {
+            dependencies {
+                implementation kotlin('test')
+                implementation kotlin('test-junit5')
+                implementation project(":packages:test-fixtures")
+            }
+        }
+
+        // ANDROID
+        androidMain {
+            dependsOn commonMain
+            dependencies {
+                implementation 'com.snowplowanalytics:snowplow-android-tracker:2.0.1@aar'
```

Comment:
@dantetam Since we use Snowplow classes in our SnowplowOptions public API this needs to be `api` instead of `implementation`. This will make it so the end user wont need to include `implementation 'com.snowplowanalytics:snowplow-android-tracker:2.0.1@aar'` in their project.